### ISSUE TITLE
`xlim(0, [1,1])` gives unhelpful error

### ISSUE THREAD

**Issue by mauro3** [JuliaGraphics/Winston.jl#197]
Accidentally using a Vector in `xlim` leads to a very cryptic error message.  Could instead a non-methods error be thrown?

```
julia> using Winston

julia> p = plot([1,2])

julia> xlim(0, [1,1])
Error showing value of type FramedPlot:
ERROR: `one` has no method matching one(::Array{Float64,1})
 in _magform at /home/mauro/.julia/v0.3/Winston/src/Winston.jl:430
 in _ticks_default_linear at /home/mauro/.julia/v0.3/Winston/src/Winston.jl:498
 in _ticks at /home/mauro/.julia/v0.3/Winston/src/Winston.jl:784
 in make at /home/mauro/.julia/v0.3/Winston/src/Winston.jl:876
 in exterior at /home/mauro/.julia/v0.3/Winston/src/Winston.jl:1228
 in interior at /home/mauro/.julia/v0.3/Winston/src/Winston.jl:1715
 in compose at /home/mauro/.julia/v0.3/Winston/src/Winston.jl:1779
 in page_compose at /home/mauro/.julia/v0.3/Winston/src/Winston.jl:1795
 in page_compose at /home/mauro/.julia/v0.3/Winston/src/Winston.jl:1783
 in anonymous at /home/mauro/.julia/v0.3/Winston/src/tk.jl:27
 in display at /home/mauro/.julia/v0.3/Winston/src/tk.jl:35
 in display at /home/mauro/.julia/v0.3/Winston/src/Winston.jl:2735
 in display at multimedia.jl:149
 in print_response at REPL.jl:139
 in print_response at REPL.jl:124
 in anonymous at REPL.jl:586
 in anonymous at REPL.jl:764
 in anonymous at LineEdit.jl:1199
 in prompt! at ./LineEdit.jl:1399
 in run_interface at ./LineEdit.jl:1374
 in run_frontend at ./REPL.jl:818
 in run_repl at ./REPL.jl:169
 in _start at ./client.jl:400
```

**Comment by mauro3** [comment]
This now throws:
```
julia> xlim(0, [1,1])                                                     
┌ Warning: Error in @guarded callback                                             
└ @ Winston ~/.julia/packages/Gtk/ADDrn/src/base.jl:92                            
ERROR: MethodError: no method matching -(::Array{Int64,1}, ::Int64)               
Closest candidates are:
  -(::Complex{Bool}, ::Real) at complex.jl:298
  -(::Missing, ::Number) at missing.jl:93
  -(::Base.CoreLogging.LogLevel, ::Integer) at logging.jl:107
  ...
```

**Comment by tknopp** [comment]
Probably one could put a `::Number` in the definition here:
https://github.com/JuliaGraphics/Winston.jl/blob/87fdbc16a772e2f376ab9cbecdced3a131d8370d/src/plot.jl#L21

**Comment by tknopp** [comment]
fixed by #296
